### ISSUE TITLE
docs: Issue in page Install ScyllaDB Linux Packages (expired apt key)

### ISSUE THREAD

**Issue by PhipBlitch-Arcadia** [scylladb/scylladb#17690]
I would like to report an issue in page https://opensource.docs.scylladb.com/branch-5.4/getting-started/install-scylla/install-on-linux

### Problem

Debian/Ubuntu install key steps reference the expired `d0a112e067426ab2`, causing package installation to fail with `The following signatures were invalid: EXPKEYSIG D0A112E067426AB2 ScyllaDB Package Signing Key 2022`

###  Suggest a fix

It [appears](https://github.com/scylladb/scylla-cluster-tests/commit/c50a6c8b26e9374f76a729a4a194e797e22fad1b) the 2024 key is `491C93B9DE7496A7`

**Comment by tzach** [comment]
@benipeled please advise

**Comment by benipeled** [comment]
Thanks @PhipBlitch-Arcadia for reporting,
We recently renewed our signing keys and generated `491C93B9DE7496A7` for future releases (greater than 5.4/2024.1 but not inclusive),
Installed keys are not aware of the renewal and are marked as expired until the local index is updated,
You can update the local index with `apt-get update` or `gpg --refresh-keys`

**Comment by mykaul** [comment]
> We recently renewed our signing keys and generated `491C93B9DE7496A7` for future releases (greater than 5.4/2024.1 but not inclusive),

I don't understand - is it applicable to 5.4/2024.1 or not?

**Comment by benipeled** [comment]
> > We recently renewed our signing keys and generated `491C93B9DE7496A7` for future releases (greater than 5.4/2024.1 but not inclusive),
> 
> I don't understand - is it applicable to 5.4/2024.1 or not?

No, it's not _"greater than 5.4/2024.1 **but not inclusive**"_,
5.4 and 2024.1 signs with `d0a112e067426ab2`

**Comment by tzach** [comment]
> No, it's not "greater than 5.4/2024.1 but not inclusive",
   5.4 and 2024.1 signs with d0a112e067426ab2

So what is the original problem? d0a112e067426ab2 was used for 5.2

**Comment by benipeled** [comment]
> > No, it's not "greater than 5.4/2024.1 but not inclusive",
> > 5.4 and 2024.1 signs with d0a112e067426ab2
> 
> So what is the original problem? d0a112e067426ab2 was used for 5.2

See above: 

> Installed keys are not aware of the renewal and are marked as expired until the local index is updated,
You can update the local index with apt-get update or gpg --refresh-keys

So in fact there is really no problem, it's all about missing local sync

**Comment by tzach** [comment]
@benipeled should we add `gpg --refresh-keys` to the instructions?
